messaging/src/main/java/org/axonframework/commandhandling/GenericCommandMessage.java

```diff
@@ -31,6 +34,7 @@
  * @author Allard Buijze
  * @since 2.0
  */
+@JsonIgnoreProperties("payloadType")
```

Comment:
Because the payloadType is already in the payload (Using the `JsonTypeInfo`). Adding this property will make Jackson throw unknown property exceptions on deserialization